db/schema.rb

```diff
@@ -547,6 +583,64 @@
     t.index ["name"], name: "index_decidim_hashtags_on_name"
   end
 
+  create_table "decidim_ideas_idea_versions", force: :cascade do |t|
```

Comment:
These seem unrelated, but I think you mentioned the migration was not checked in, so I think it's fine to leave in this PR.